src/poliastro/plotting.py

```diff
@@ -375,3 +374,19 @@ def savefig(self, filename, **layout_kwargs):
             image=ext[1:], image_filename=basename,
             show_link=False,  # Boilerplate
         )
+
+
+def plot_solar_system(epoch=None):
+    orbits = []
+    orbits.append(Orbit.from_body_ephem(Earth, epoch))
```

Comment:
Good use of a `for` loop to call `op.plot` repeatedly. Can yo udo the same for `from_body_ephem`? There's a way to write this function in four or five lines :wink:

Reply:
Actually Yes! But then I have to import classes.I will do that!